resources/servo.css

```diff
@@ -7,6 +7,7 @@ input[type="reset"]     { background: lightgrey; border-top: solid 1px #EEEEEE;
 input[type="hidden"]    { display: none !important }
 input[type="checkbox"],
 input[type="radio"]     { font-family: monospace !important; border: none !important; background: transparent; }
+input[dir="rtl"]        { text-align: right; }
```

Comment:
Oh, this isn't the same as the [dir selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir). This is explicitly matching `<input dir="rtl">`, which is different.